ui/hooks/accounts/useMultichainClient.ts

```diff
@@ -7,26 +7,46 @@ import {
   multichainUpdateBalance,
 } from '../../store/actions';
 import { BITCOIN_WALLET_SNAP_ID } from '../../../shared/lib/accounts/bitcoin-wallet-snap';
+import { SOLANA_WALLET_SNAP_ID } from '../../../shared/lib/accounts/solana-wallet-snap';
+
+export enum WalletClientType {
+  Bitcoin = 'bitcoin-wallet-snap',
+  Solana = 'solana-wallet-snap',
+}
+const SNAP_ID_MAP: Record<WalletClientType, string> = {
```

Comment:
Should this be `SnapId` instead?
```suggestion

const SNAP_ID_MAP: Record<WalletClientType, SnapId> = {
```

Reply:
Nit: Also, `SNAP_ID_MAP` seems to generic, what about `WALLET_SNAP_ID_MAP`?